descriptors/ModuleDescriptor-template.json

```diff
@@ -221,6 +221,22 @@
         }
       ]
     },
+    {
+      "id": "patron-notice-policy-storage",
+      "version": "1.0",
```

Comment:
Given that this interface is very partial at the moment (as the collection and item GET endpoints have been separated out), I think it is worth considering using a 0.x version here.

For context, previous conversation in FOLIO agreed that an interface version is effectively published when a change is merged to the master branch of the implementing module. This means that the subsequent work (CIRCSTORE-90, CIRCSTORE-91, CIRCSTORE-92 etc all need to change the interface version as well.

Does that make sense?